tests/unit_tests/test_config.py

```
@@ -1,6 +1,216 @@
+        assert ff.FAKE_LOGIN is True
+
+    def test_getattr_lowercase(self) -> None:
+        """Test attribute access by lowercase name."""
+        ff = FeatureFlags((FakeLogin,), {"FAKE_LOGIN": True})
+        assert ff.fake_login is True
+
+    def test_getattr_missing_raises(self) -> None:
+        """Test attribute access for missing flag raises AttributeError."""
+        ff = FeatureFlags((FakeLogin,), {})
+        with pytest.raises(AttributeError, match="no flag"):
+            ff.nonexistent
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)